snuba/utils/streams/batching.py

```diff
@@ -14,25 +14,31 @@
 )
 
 from snuba.utils.metrics.backends.abstract import MetricsBackend
-from snuba.utils.streams.abstract import Consumer, ConsumerError, Message
-from snuba.utils.streams.kafka import TopicPartition
+from snuba.utils.streams.abstract import (
+    Consumer,
+    ConsumerError,
+    Message,
+    TStream,
+    TOffset,
+    TValue,
+)
 
 
-logger = logging.getLogger("batching-kafka-consumer")
+logger = logging.getLogger(__name__)
 
 TMessage = TypeVar("TMessage")
 
 
 class AbstractBatchWorker(ABC, Generic[TMessage]):
-    """The `BatchingKafkaConsumer` requires an instance of this class to
+    """The `BatchingConsumer` requires an instance of this class to
     handle user provided work such as processing raw messages and flushing
     processed batches to a custom backend."""
 
     @abstractmethod
     def process_message(self, message: TMessage) -> Optional[Any]:
```

Comment:
Is the return type going to remain Any or is this going to become a second parameter for the generic ?

Reply:
Doing that as a follow up (like right now)